src/field/index.js

```diff
@@ -1,4 +1,5 @@
 import ReactDOM from 'react-dom';
+import { get } from 'lodash';
```

Comment:
We don't have a plan to make `lodash` be our dependencies, it's better to write a `get` method in util

Reply:
@youluna Resolved